lib/mhv_ac/configuration.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+require 'common/client/configuration/rest'
+require 'common/client/middleware/request/camelcase'
+require 'common/client/middleware/response/json_parser'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/mhv_errors'
+require 'common/client/middleware/response/snakecase'
+
+module MHVAC
+  # Configuration class used to setup the environment used by client
+  class Configuration < Common::Client::Configuration::REST
+    def app_token
+      Settings.mhv.rx.app_token
+    end
+
+    def base_path
+      "#{Settings.mhv.rx.host}/mhv-api/patient/v1/"
+    end
+
+    def service_name
+      'MHVAC'
```

Comment:
I see the Emis service is using more descriptive, longer service names. I think something like "MHV_Acct" might be less obscure to somebody looking at breakers in the future.